### PR TITLE
Rename '--config' to '--extra-settings' for clarity and avoid conflicts

### PR DESCRIPTION
**Pull request by PeterJCLaw**
This works around `django-configurations` interception of arguments which are contractions of its `--configuration` argument (https://github.com/jazzband/django-configurations/issues/343).

We also take the opportunity to slightly clarify the name of this argument.

Have tested locally that the impacted commands still behave as expected using the new spelling of the argument.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -149,10 +149,13 @@ part of a pool:
 $ python manage.py queue_runner --machine 2 --of 4
 ```
 
-Alternatively a runner can be told explicitly which configuration to use:
+Alternatively a runner can have be told explicitly how to be have by having
```
Comment (lirsacc) [inline README.md:152]: Is that supposed to be:

```suggestion
Alternatively a runner can be told explicitly how to behave by having
```

**Review by lirsacc — APPROVED** [review]
One thought: from the behaviour should this be named something closer to `overrides` to make it clear it's not just additive? (Don't feel too strongly about it though)

**Comment by PeterJCLaw** [comment]
> One thought: from the behaviour should this be named something closer to `overrides` to make it clear it's not just additive?

Yeah, that might be better. I expect it would need to be `--override-settings` or similar rather than the more natural `--settings-overrides` to avoid clashes with Django's built-in `--settings` flag that `django-configurations` behaviour would similarly mangle.

Will revisit this before merging tomorrow.

**Comment by PeterJCLaw** [comment]
I've improved the inline documentation of the CLI arguments, but I'm going to stick with the name for now.

**Pull request merged**